packages/generator/src/operations/import.ts

```diff
@@ -117,29 +124,41 @@ export function operationImportDeclarations(
         }
       ];
 
+  const namedImports = [
+    ...edmRelatedImports(returnTypes),
+    ...complexTypeRelatedImports(returnTypes),
+    ...responseTransformerImports(returnTypes),
+    'DeSerializers',
+    'DefaultDeSerializers',
+    'defaultDeSerializers',
+    ...propertyTypeImportNames(parameters)
```

Comment:
[req] The lines below are a clear use case of the tenary operator:
```ts
includesUnbound && hasFunction ? ['FunctionImportRequestBuilder'] : []
```

or which I would find even better, create a little helper function `operationRelatedImport(operation)` which returns a list of string, and you move the code above and below in this helper function and use it like the other methods above `...responseTransformerImports`

Reply:
Thanks for the suggestion! I will move these to a helper function as suggested.